_opcodes/op67-bgee.html

```diff
@@ -0,0 +1,56 @@
+---
+n: 67
+name: "Summon: Creature Summoning"
+param1: "Unknown"
+param2: "Control"
+bg1: 0
+bg2: 0
+bgee: 1
+iwd1: 0
+iwd2: 0
+pst: 0
+---
+Summons a creature, with EA set to the 'Control' field. The Resource key should be set to the filename of the creature to summon.
+<br/>Known values for 'Control' are:
+<ul>
+	<li>0 &longrightarrow; Match target (hostile if no target)</li>
+	<li>1 &longrightarrow; Match target (hostile if no target)</li>
+	<li>2 &longrightarrow; As creature file</li>
+	<li>3 &longrightarrow; Match target (hostile if no target)</li>
+	<li>4 &longrightarrow; As creature file</li>
+	<li>5 &longrightarrow; Hostile to target</li>
+	<li>6 &longrightarrow; As creature file</li>
+	<li>8 &longrightarrow; As creature file</li>
+	<li>255 &longrightarrow; As creature file</li>
+</ul>
+
+<br/><br/>If effect's target is set as 'Original Caster' then the 'Control' parameter covers 'Location' as well as control. In this case, known values for 'Control' are:
+<ul>
+	<li>2 &longrightarrow; Summoned at location of target creature</li>
+	<li>4 &longrightarrow; Summoned at location of caster</li>
+	<li>6 &longrightarrow; Summoned at location of caster</li>
+</ul>
+
+{% capture note %}
+<ul>
+	<li>'Control' values 0 and 1 are identical, 4 and 6 are identical, 3 and 5 are each unique, all other values are identical with each other.</li>
+	<li>'Control' values 3, 4, 5, and 6 summon the creature at the location of the effect target. All other values summon the creature at the location of the ability target.</li>
+</ul>
+{% endcapture %}
+
+{% include note.html %}
+
+{% capture note %}
+<ul>
+	<li>"Resource2" (external <a href="../file_formats/ie_formats/eff_v2.htm">EFF</a>s): Animation VVC/BAM file to be played at the summoned creature's location.<br />
+		Default is a hardcoded VVC using "SPBOOM.BAM" and palette #65. This is generally used as the "portal" animation.
+	</li>
+	<li>"Resource3" (external <a href="../file_formats/ie_formats/eff_v2.htm">EFF</a>s): Animation VVC/BAM file to be played at the location of the ability target.</li>
```

Comment:
This is the only new thing I see in the EE version, so I suggest we merge them back.

Reply:
Done.